GoogleForCreators/web-stories-wp
# Date Picker: show timezone abbreviation if possible

## Context



## Summary

Have a more human readable time zone in date picker. 

## Relevant Technical Choices



## To-do



## User-facing changes

<img width="449" alt="Screenshot 2022-09-21 at 16 40 39" src="https://user-images.githubusercontent.com/237508/191549374-81190dd4-1730-4616-826f-6cd58352e5c2.png">


## Testing Instructions






This PR can be tested by following these steps:

1. Go to Settings > General. 
2. Change time zone to berlin. 
3. Create a new story. 
4. Click on the documents tab. 
5. Click on publish date ( immediately ). 
6. See time zone. 
7. Hover over time zone. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12234
